scalability_and_performance/planning-your-environment-according-to-object-maximums.adoc

```diff
@@ -0,0 +1,22 @@
+git [id="planning-your-environment-according-to-object-maximums"]
+= Planning your environment according to object maximums
+include::modules/common-attributes.adoc[]
+:context: object-limits
+
+toc::[]
+
+Consider the following tested object maximums when you plan your {product-title} cluster.
+
+These guidelines are based on the largest possible cluster. For smaller
+clusters, the maximums are proportionally lower. There are many factors that
```

Comment:
I would remove "proportionally". Because it is not necessarily true that there is a linear relationship and therefore for smaller clusters the numbers will be proportionally lower.